src/autopas/tuning/tuningStrategy/fuzzyTuning/FuzzyTuning.cpp

```diff
@@ -0,0 +1,288 @@
+    });
+  }
+
+  AutoPasLog(DEBUG, "Fuzzy tuning selected {} out of {} possible configurations: {}", newSearchSpace.size(),
+             configQueue.size(), [&newSearchSpace]() {
+               std::string result;
+               for (const auto &config : newSearchSpace) {
+                 result += config.toString() + ", ";
+               }
+               return result;
+             }());
+  configQueue.clear();
+  std::copy(newSearchSpace.rbegin(), newSearchSpace.rend(), std::back_inserter(configQueue));
+}
+
+void FuzzyTuning::updateQueueInterpretOutputAsSuitability(std::vector<Configuration> &configQueue) {
+  using namespace autopas::fuzzy_logic;
+
+  std::list<Configuration> newSearchSpace{configQueue.begin(), configQueue.end()};
+
+  std::vector<std::pair<ConfigurationPattern, double>> configSuitabilities;
+
+  // For each fuzzy control system
+  for (const auto &[name, fcs] : _fuzzyControlSystems) {
+    // 1. Predict
+    auto prediction = fcs->predict(_currentLiveInfo);
+
+    // 2. Get single configuration pattern for each system
+    const auto &configurations = _outputMappings.at(name)->getClosestConfigurationPatterns(prediction);
+    if (configurations.size() != 1) {
+      utils::ExceptionHandler::exception(
+          "FuzzyTuning: Expected exactly one configuration pattern for suitability "
+          "value, but got {}. In system {} with prediction {}",
+          configurations.size(), name, prediction);
+    }
+
+    // 3. Insert suitability values
+    configSuitabilities.emplace_back(configurations[0], prediction);
+
+    AutoPasLog(DEBUG, "FuzzySystem predicted suitability {} for pattern {}", prediction, configurations[0].toString());
+  }
+
+  // Sort the suitability values
+  std::sort(configSuitabilities.begin(), configSuitabilities.end(),
+            [](const auto &a, const auto &b) { return a.second > b.second; });
+
+  // get the best configurations and everything that is within 10% of the best
+  const double THRESHOLD = std::stod(_fuzzyControlSettings->at("suitabilityThreshold"));
+
+  const auto bestSuitability = configSuitabilities.front().second;
+  std::vector<ConfigurationPattern> bestConfigurations;
+  for (const auto &[pattern, suitability] : configSuitabilities) {
+    if (suitability < bestSuitability * (1 - THRESHOLD)) {
+      break;
+    }
+
+    AutoPasLog(DEBUG,
+               "FuzzyTuning: Adding configuration pattern {} with suitability {} since its within {}% of the "
+               "best suitability {}",
+               pattern.toString(), suitability, THRESHOLD * 100, bestSuitability);
+    bestConfigurations.push_back(pattern);
+  }
+
+  // 4. Only keep configurations that are in the best configurations
+  newSearchSpace.remove_if([&bestConfigurations](const Configuration &configuration) {
+    return std::none_of(bestConfigurations.begin(), bestConfigurations.end(),
+                        [&configuration](const auto &pattern) { return pattern.matches(configuration); });
```

Comment:
This could probably be changed, yes. However, I wanted to stay consistent with the `RuleBasedTuning` Strategy as it is done similarly there. 
Performance wise it is irrelevant, as this only gets executed once per tuning phase so it is more of code-style issue. 
So I would suggest leaving it that way to stay consistent with `RuleBasedTuning` (or change both of them).